DFE-Digital/itt-mentor-services
# Import provider data

## Context

Rather than rely on the Publish API, we are importing Provider data into our application.

## Changes proposed in this pull request

- Add an importer for generating 

## Guidance to review

_Test Importer_
- run `bundle exec rake db:drop`
- run `bundle exec rake db:create`
- run `bundle exec rake db:setup`
- run `rails c`
- run `Provider.count` (_should return 184 records_)
- run `exit` (exit rails console)

_Test Add Organisation (Placement::Provider)_
- Sign in as Colin
- Click "Add Organisation"
- Select "ITT Provider" radio button
- Click "Continue"
- Type a few letters into the text input (e.g "Bath")
- Select an option from the dropdown list.
- Click "Continue"
- This should take you to the "Check your answers" page
- Click "Add organisation"
- Provider Name should appear in the Organisation List
(Feel free to test for error handling)

## Link to Trello card



## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
